backend/app/api/routes/credentials.py

```
@@ -60,95 +55,124 @@ def create_new_credential(*, session: SessionDep, creds_in: CredsCreate):
-    new_creds = set_creds_for_org(session=session, creds_add=creds_in)
-    if not new_creds:
+    credential = set_creds_for_org(
+        session=session,
+        creds_add=creds_in,
+        organization_id=_current_user.organization_id,
+        project_id=_current_user.project_id,
+    )
+    if not credential:
         raise Exception(status_code=500, detail="Failed to create credentials")
 
     return APIResponse.success_response([cred.to_public() for cred in new_creds])
```

Ruff (0.12.2)

Undefined name `new_creds`

(F821)